src/main/java/com/acme/dbo/txlog/Facade.java

```diff
@@ -1,54 +1,52 @@
 package com.acme.dbo.txlog;
 
-import javafx.util.Pair;
-
-import java.util.Arrays;
-
-import static java.lang.Math.abs;
-
 public class Facade {
-    private static Boolean isDecorated=true;
 
     public static void log(int message) {
-    Controller.logInteger(message,  isDecorated);
+        Controller.intCommand = new IntCommand(message);
+        Controller.log(Controller.intCommand);
     }
 
-    public static void log(boolean message) { Controller.logBoolean(message,  isDecorated); }
-
     public static void log(byte message) {
-        Controller.logByte(message,  isDecorated);
+        Controller.byteCommand = new ByteCommand(message);
+        Controller.log(Controller.byteCommand);
```

Comment:
maybe inline variable? There is no value in separate variable so you can inline it:
`Controller.log(new ByteCommand());`

Reply:
Would there be a link in this case inside Controller to ByteCommand?

Reply:
It depends on Controller's logic. If it decides to store it in variable or not (based on business logic of accumulating/flushing). From the outside we only have to give command and not to manage Controller's internal state. It have to be managed by Controller only.

Reply:
Done